index.html

```diff
@@ -1632,7 +1632,6 @@ <h4>
         If the |recordType| value is "`smart-poster`", or if running
         <a>validate external type</a> on |recordType| returns `true`,
         then return the result of running <a>parse records from bytes</a> on |bytes|.
-        Re-[= exception/throw =] any exceptions.
```

Comment:
Instead of re-throwing, should we return `null`?

Reply:
We're already returning `null`.